src/Animations/babylon.animation.ts

```diff
@@ -1,8 +1,21 @@
 ﻿module BABYLON {
+    /**
+     * Represents the range of an animation.
+     */
     export class AnimationRange {
+        /**
+         * Initializes the range of an animation.
+         * @param {string} name - The name of the animation range.
+         * @param {number} from - The starting frame of the animation.
+         * @param {number} to - The ending frame of the animation.
+         */
         constructor(public name: string, public from: number, public to: number) {

```

Comment:
You can run gulp typedoc-check to do a typedoc check
Here you need to also comment parameters as they are public

Reply:
Fixed